routers/imagesRouter.js

```diff
@@ -34,6 +34,21 @@ router.get('/:imageName', (req, res) => {
   })
 })
 
+router.post('/upload', adminAction, (req, res) => {
```

Comment:
Please explain what you did here

Reply:
I didn't notice that the existing post was supposed to do this. Should I combine them?